acceptance/tests/ensure_puppet-agent_paths.rb

```diff
@@ -90,17 +114,37 @@
   :posix => ['puppet', 'facter', 'hiera', 'mco', 'cfacter'],
   :win   => ['puppet.bat', 'facter.bat', 'hiera.bat', 'cfacter.bat']
 }
-locations = {:posix => {:aio => '/opt/puppetlabs/bin',                     :git => '/usr/bin'},
-             :win   => {:aio => 'C:/Program Files/Puppet Labs/Puppet/bin', :git => '/usr/bin'}}
+
+def locations(platform, ruby_arch, type)
+  if type != 'aio'
+    return '/usr/bin'
+  end
+
+  case platform
+  when /windows/
+    # If undefined, ruby_arch defaults to x86
+    if ruby_arch == 'x64'
+      ruby_arch = /-64/
+    else
+      ruby_arch = /-32/
```

Comment:
It's valid, quick test: `'windows-2003r2-64' =~ /-2003/`.

It's only a keyword in a square-bracket expression, such as /[2-3]/.